ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -113,6 +113,20 @@ tests:
     - ref: openshift-qe-perfscale-aws-data-path-sg
     - chain: openshift-qe-data-path-tests
   timeout: 5h0m0s
+- as: perfscale-aws-ovn-ipsec-control-plane-120nodes-periodic
+  cron: 0 15 * * 2
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      SIZE_VARIANT: large
+    test:
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: openshift-qe-control-plane
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
```

Comment:
Should we not just call ipi-install w/ 

```
    pre:
    - chain: ipi-conf-aws
    - ref: ovn-conf
    - ref: ovn-conf-ipsec-manifest
    - chain: ipi-install
```

Reply:
@jtaleric QE had some test profiles that are running on our Jenkins CI, we are planing to move them to prow. Those profiles includes some configurations like IPSEC, FIPS, Private cluster config on different providers, like the list in https://docs.google.com/spreadsheets/d/1TM73n4Y6zKRQjvCX8zFM0LwYxChdsncTEGPHpdsS9Ig/edit#gid=1709312908. OpenShift functional QE team already had mature workflows for those installation configurations like 'cucushift-installer-rehearse-aws-ipi-ovn-ipsec'. So we hope to make use of them instead of doing again ourselves. That will be similar to our rosa installation that uses existing workflow: rosa-aws-sts. 
This pr is to try to see if there is any problem using the existing workflow like 'cucushift-installer-rehearse-aws-ipi-ovn-ipsec'. I added a task link in the first comment. We can further discuss this if you have any comments or concern.